release.nix

```diff
@@ -0,0 +1,25 @@
+let
+
+  config = {
+    packageOverrides = pkgs: rec {
+      haskellPackages = pkgs.haskell.packages.ghc822.override {
+        overrides = haskellPackagesNew: haskellPackagesOld: rec {
+          universum           = pkgs.haskell.lib.dontCheck (haskellPackagesNew.callHackage "universum" "1.1.0" {});
+          # LIB
+          cardano-shell  = haskellPackagesNew.callPackage ./cardano-shell.nix { }; 
+          cardano-chain-env = pkgs.haskell.lib.dontCheck (haskellPackagesNew.lib.buildStackProject { 
```

Comment:
We don't need this and I have no clue how this is supposed to work like this.

Reply:
I believe release.nix is required by Hydra.